code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -173,15 +173,23 @@
 \***************************************/
 /datum/action/changeling/weapon/arm_blade
 	name = "Arm Blade"
-	desc = "We reform one of our arms into a deadly blade. Costs 20 chemicals."
-	helptext = "We may retract our armblade in the same manner as we form it. Cannot be used while in lesser form."
+	desc = "We reform one of our arms into a deadly blade that breaks after a number of hits, improvable by absorbing genomes. Costs 40 chemicals."
+	helptext = "We may retract our armblade in the same manner as we form it. Organic tissue is not perfect; the armblade will break after it is used too much. The more genomes we absorb, the stronger it is. Cannot be used while in lesser form."
 	button_icon_state = "armblade"
-	chemical_cost = 20
-	dna_cost = 2
+	chemical_cost = 40
+	dna_cost = 4
```

Comment:
4 DNA cost is way too much, especially since you're making it less usable. Keep it at 2

Likewise you're making armblade disposable but upping the usecost, why? Keeping it where it is is fine, since they have to re-use it to get a new one often.